### PR TITLE
Issues #13098 - ZPLSC-C Parser: Added Individual Colorbars For Each Fequency's Echogram

### PR DESCRIPTION
**Pull request by renegelinas**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# Version <Next Release>
+
+* Issue #13098 - Added individual colorbars for each fequency's echogram.
```
Comment (danmergens) [inline RELEASE_NOTES.md]: `frequency`

File: mi/common/zpls_plot.py [hunk 2]
```diff
@@ -0,0 +1,178 @@
+"""
+@package mi.common
+@file mi/common/zpls_plot.py
+@author Rene Gelinas
+@brief ZPLSC Echogram generation for the ooicore
+
+Release notes:
+
+This class supports the generation of ZPLS echograms. It needs matplotlib version 1.3.1 for the code to display
```
Comment (danmergens) [inline mi/common/zpls_plot.py]: Please update requirements.txt to include the correct matplotlib version.

File: RELEASE_NOTES.md (hunk 1 above)
Comment (renegelinas) [reply RELEASE_NOTES.md]: Done.

File: mi/common/zpls_plot.py (hunk 2 above)
Comment (renegelinas) [reply mi/common/zpls_plot.py]: Done.
Updated the text, "matplotlib version (or higher)".
Updated the requirements.txt to reflect the same.

**Comment by renegelinas** [comment]
Thanks Dan

**Pull request merged**